openwisp_radius/api/views.py

```diff
@@ -101,16 +102,24 @@ def _get_ip_list(self, uuid):
                 ip_list = None
             else:
                 cache.set(f'ip-{uuid}', ip_list)
-        return ip_list
+        return ip_list if ip_list else app_settings.FREERADIUS_ALLOWED_HOSTS
```

Comment:
this could be simply:

`return ip_list or app_settings.FREERADIUS_ALLOWED_HOSTS`

If `ip_list` is falsy, the second value will be returned.

Reply:
Great, looks better.